docs/pages/guides/webhooks.mdx

```diff
@@ -2,98 +2,296 @@
 meta:
   title: "Webhooks"
   parentTitle: "Guides"
-  description: "Learn Liveblocks Webhooks concepts and implementation"
+  description: "Learn Liveblocks webhooks concepts and implementation"
 ---
 
 In this guide, we will explain how to use webhooks within your product so that
 you can react to Liveblocks events as they happen. Now that our platform is
 observable, you can use webhooks to reduce development time and the need for
 polling.
 
-## Why Webhooks
+## Configuring webhooks
 
-Webhooks connect two applications by letting one subscribe to events created by
-the other. The listening application (yours) subscribes to events, and the event
-emitting application (Liveblocks) submits events via HTTP requests to the
-listening application's URL. Thus, using webhooks with Liveblocks reduces the
-need to poll Liveblocks data as you are now alerted when the data changes.
-Leveraging webhooks makes Liveblocks observable and reduces the need for cron
-jobs to poll Liveblocks data. As a result, your application (the listening
-application) will have less complex logic related to consuming data.
+To set up webhooks for your project, you’ll need to create an endpoint,
+subscribe to events, and secure your endpoint.
 
-There are many scenarios where webhooks could benefit your application. For
-example, you can use webhooks to monitor when users enter and leave a room.
-After some time has passed, you could create logic that syncs Liveblocks storage
-to your application database if no users are remaining.
+- [Creating an endpoint](#creating-an-endpoint)
+- [Subscribing to events](#subscribing-to-events)
+- [Security verification](#security-verification)
+- [Replaying events](#replaying-events)
 
-You could also use webhooks to display how many users are currently active in a
-room at any given time. The possibilities are endless, and as time progresses,
-we will include more events for your application to subscribe to.
+### Creating an endpoint [#creating-an-endpoint]
 
-## What is a webhook?
+If you would like to create an endpoint to receive webhook events, you will do
+so from within the webhooks dashboard for your project.
 
-Webhooks pass information from one application to another instantaneously. A
-webhook is simply an HTTP callback triggered by an event. When an application
-subscribes to an event, and that event occurs, the webhook invokes the HTTP
-callback with a URL you specify (called the webhook endpoint). Webhooks
-streamline the collection of data between two connected services in real-time.
+1. From the [dashboard](/dashboard) overview, navigate to the project you would
```

Comment:
Not new to this PR, but we should find a way to have those links open in a new tab not to break navigation.
Maybe a solution exists? See [this thread](https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown)


cc @pierrelevaillant

Reply:
`<a href="/dashboard" target="_blank">…</a>`